templates/functionaries.html

```diff
@@ -0,0 +1,126 @@
+{#-#################################################################
+<File Name>
+  functionaries.html
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Jun 30, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Used to add new functionaries (by name) and upload functionary public key
+  using a file upload dropzone for each newly added functionary.
+
+#################################################################-#}
+{% import '_macros.html' as macros %}
+{% extends "base.html" %}
+
+{#- BEGIN: Functionary form macro -#}
+{% macro item_form(name=None, template=False) %}
+<div class="functionary dz-container p-3 mt-2 {{'template' if template}}">
+  <div class="m-2">
+    <span class="functionary-name">{{name}}</span><br>
+    <small>(Drop public key or click in box to upload)</small>
+  </div>
+  <form class="dropzone m-2 w-100" method="POST", enctype="multipart/form-data"
+      action="{{ url_for('ajax_upload_key') }}">
+    <input type="hidden" name="functionary_name" value="{{name}}">
+  </form>
+  <button type="button" class="rm-func-btn btn btn-outline-danger m-2">Remove Functionary</button>
+</div>
+{% endmacro %}
+{#- END: Functionary form macro -#}
+
+
+{#- BEGIN: Rendered content -#}
+{% block content %}
+  {{ macros.progress_bar(value=65) }}
+  <hr>
+  <h1>The who-is-who of your software supply chain</h1>
+  <p>An essential part of <i>in-toto</i> is that you can verifiably specify who
+  is allowed to do what in your project. The actors that are allowed to perform
+  a step or multiple steps in your software supply chain are called
+  <i>functionaries</i> and are identified by their respective public key.
+  Below you can define your functionaries and upload their public key.</p>
```

Comment:
Consider stating which key formats are supported.

Reply:
See https://github.com/in-toto/layout-web-tool/issues/26#issuecomment-329889199